docs/reference/configuration.rst

```diff
@@ -643,6 +643,33 @@ The path to the media root of the cms media files.
 
 .. setting:: CMS_MEDIA_URL
 
+
+CMS_UNESCAPED_RENDER_MODEL_TAGS
+===============================
+
+default
+    True
+
+.. warning::
+    In this version of django CMS, this setting has a default value of
+    ``True`` to provide behavior consistent with previous releases.
+    However, all developers are encouraged to set this value to ``False`` to
+    help prevent a range of security vulnerabilities stemming from
+    HTML, Javascript, and CSS Code Injection.
+
+.. important::
```

Comment:
Advise here that when removed, the behavior will be the opposite of the current default